tests/codemods/test_secure_flask_cookie.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from core_codemods.secure_flask_cookie import SecureFlaskCookie
+from tests.codemods.base_codemod_test import BaseSemgrepCodemodTest
+from textwrap import dedent
+
+each_func = pytest.mark.parametrize("func", ["make_response", "Response"])
+
+
+class TestSecureFlaskCookie(BaseSemgrepCodemodTest):
+    codemod = SecureFlaskCookie
+
+    def test_name(self):
+        assert self.codemod.name() == "secure-flask-cookie"
+
+    @each_func
+    def test_import(self, tmpdir, func):
+        input_code = f"""\
+        import flask
+
+        response = flask.{func}()
+        var = "hello"
+        response.set_cookie("name", "value")
+        """
+        expexted_output = f"""\
+        import flask
+
+        response = flask.{func}()
+        var = "hello"
+        response.set_cookie("name", "value", secure=True, httponly=True, samesite='Lax')
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expexted_output))
+
+    @each_func
+    def test_from_import(self, tmpdir, func):
+        input_code = f"""\
+        from flask import {func}
+
+        response = {func}()
+        var = "hello"
+        response.set_cookie("name", "value")
+        """
+        expexted_output = f"""\
+        from flask import {func}
+
+        response = {func}()
+        var = "hello"
+        response.set_cookie("name", "value", secure=True, httponly=True, samesite='Lax')
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expexted_output))
+
+    @each_func
+    def test_import_alias(self, tmpdir, func):
+        input_code = f"""\
+        from flask import {func} as flask_resp
+        var = "hello"
+        flask_resp().set_cookie("name", "value")
+        """
+        expexted_output = f"""\
+        from flask import {func} as flask_resp
+        var = "hello"
+        flask_resp().set_cookie("name", "value", secure=True, httponly=True, samesite='Lax')
+        """
+        self.run_and_assert(tmpdir, dedent(input_code), dedent(expexted_output))
+
+    @pytest.mark.parametrize(
+        "input_args,expected_args",
+        [
+            (
+                "secure=True",
+                "secure=True, httponly=True, samesite='Lax'",
+            ),
+            (
+                "secure=True, httponly=True, samesite='Lax'",
+                "secure=True, httponly=True, samesite='Lax'",
+            ),
+            (
+                "secure=True, httponly=True, samesite='Strict'",
+                "secure=True, httponly=True, samesite='Lax'",
```

Comment:
We could consider allowing users to pass samesite=Strict and not changing it to Lax

Reply:
Yes I think we should not modify `samesite` if it is already less permissive than our codemod.